### PR TITLE
Project news site 

### PR DESCRIPTION
**Pull request by efstasia**
https://loving-goldberg-e49bb3.netlify.app/

I'm very proud of trying both grid and flexbox. I'm also very proud of the layout and the responsiveness. 
In the beginning I struggled with the layout and aligning everything, but I managed to fix that by starting all over and eventually creating the webpage two times with both flexbox and grid.

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,4 +1,215 @@
...
+      opacity: 0;
+  }
+ 
+  100% {
+      transform: translateY(0%);
+      opacity: 1;
+  }
+}
+
+@keyframes start {
```
Comment (Ajliin) [inline code/style.css:140]: Same thing as before - add a comment on how it works would help (maybe also for yourself and remember?)

**Review by Ajliin — COMMENTED** [review]
Such a brilliant job! All requirements are met, and the site looks super nice (and I love the cats of course!) Up to black level, well done!

You also made the extras - with links that navigate you to the right place in the site. As I written in the document, it would be nice with some comments when you have done things outside scope, to make it a bit easier for reviewer?

Very good structure overall, it is easy to find things.

Maybe next time try mobile first approach?

I have learned so much by reviewing your code, thank you for doing so much of the extra so I have learned new things!

File: code/index.html [hunk 2]
```diff
@@ -4,19 +4,108 @@
...
+
+<body>
+    <header class="header">
+      <img class="logo" src="images/cat.png" alt="logo of black cat sitting"/>
+        <h1>cat-astrophe</h1>
+      <nav class="navigation">
+        <a href="#big-news">Big news</a>
+        <a href="#small-news">Small news</a>
+        <a target=”_blank”  href="https://whiskerstotailspetsitting.com/2017/05/purr-fect-list-cat-vocabulary-not-kitten-around/">More puns</a>
+        <a href="#">Contact</a>
```
[the remark this answers is not in the document]
Comment (efstasia) [reply code/index.html:19]: Will write a comment about it being just for show and non click able!

File: code/index.html [hunk 3]
```diff
@@ -4,19 +4,108 @@
...
+          <h2><strong>It's a cats world</strong></h2>
+          <p>“I am what I am. I would tell you what you want to know if I could, for you have been kind to me. But I am a cat, and no cat anywhere ever gave anyone a straight answer.”</p> 
+          <p><strong>― S. Beagle, The Last Unicorn</strong></p>
+        </div>
+      </section>
 
-    <section class="">
-      <!-- The other news  -->
+<!---small news-->
+  <div id="small-news"></div>
```
[the remark this answers is not in the document]
Comment (efstasia) [reply code/index.html:38]: Thank you! Will add a comment ☺

File: code/index.html [hunk 4]
```diff
@@ -4,19 +4,108 @@
...
+          <p><strong><em>noun</em></strong></p>
+          <p>1. The capacity or ability to direct or influence the behavior of other cats.</p>
+          <p>2.“The idea that men should have paw-er over cats is absurd. Come feed me, now.”</p>
+        </div> 
     </section>
-  </body>
+</body>
+
+<footer>
+  <div class="links">
```
[the remark this answers is not in the document]
Comment (efstasia) [reply code/index.html:100]: Didn't even think of connecting the "contact" with the footer, good idea!

File: code/style.css [hunk 5]
```diff
@@ -1,4 +1,215 @@
...
+.navigation a {
+  color: black;
+  margin: 0 10px;
+  font-size: 1.2em;
+  text-decoration: none;
+  background-color: lightgray;
+}
+
+
+a:visited {
```
[the remark this answers is not in the document]
Comment (efstasia) [reply code/style.css:36]: Links by default turns blue when you have clicked them once and I wanted to keep them black

File: code/style.css [hunk 6]
```diff
@@ -1,4 +1,215 @@
...
+  animation: start 2s ease-in-out;
+}
+
+.big-news-img {
+  max-width: 90%;
+  box-shadow: 4px 6px 6px rgba(0, 0, 0, 0.8);
+  margin: 5px 0 10px 5px;
+}
+
+.big-news-img:hover {
```
[the remark this answers is not in the document]
Comment (efstasia) [reply code/style.css:64]: Good to know!

File: code/style.css (hunk 1 above)
Comment (efstasia) [reply code/style.css:140]: Will add comment!

**Pull request closed without merging**